lib/registries/addon/components/registries-metadata/component.ts

```diff
@@ -22,9 +26,16 @@ export default class RegistriesMetadata extends Component {
     registration?: Registration;
     extendedFields?: boolean;
 
+    @tracked provider?: RegistrationProviderModel;
     // Private properties
     expandCitations = false;
 
+    @task({ on: 'didReceiveAttrs'})
+    @waitFor
+    async fetchProvider() {
+        this.provider = await this.registration?.provider;
```

Comment:
Would this be better as:

```
if this.registration:
    this.provider = await this.registration.provider;
```

Or are you quite sure that you'll have the registration since you're `on: 'didReceiveAttrs'` and you're on a registration page, that you just need to make typescript happy?

Reply:
I made the assumption that there will be a registration, so this is more to appease typescript